zubhub_frontend/zubhub/public/locales/en/translation.json

```diff
@@ -240,11 +240,14 @@
     "allProjects": "All Projects",
     "prev": "Prev",
     "next": "Next",
+    "createProject": "Create a project",
+    "sorry": "Sorry",
```

Comment:
Since 100% of people installing a new Zubhub will be landing on this page, maybe we make it more "happy"  and not an "error" page. Lets call it `Welcome`